invenio_records_ui/permissions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
IMHO we can remove the reference to `invenio_records` and replace it with `perm_factory` defined in the paragraph above.